### DISCUSSION TITLE
Error 404 for image file (via pages) which exists in repository

### DISCUSSION THREAD

**Discussion by svkucheryavski** [community/community#21838] [Other Feature Feedback, Questions, & Ideas]
<p>I have a rather strange issue. There are two repositories, each has “docs” directory with content hosted via GitHub pages. I use a custom domain for that. However, some files (mostly images) are not accessable from one repo although they are visible from another one. Here is an example:</p>
<p>Raw file from one repo:</p>
<p><a href="https://github.com/svkucheryavski/mdatools/blob/master/docs/_main_files/figure-html/unnamed-chunk-10-1.png" rel="nofollow noopener">https://github.com/svkucheryavski/mdatools/blob/master/docs/_main_files/figure-html/unnamed-chunk-10-1.png</a></p>
<p>This file via Git Hub Pages:</p>
<p><a href="http://mdatools.com/mdatools/_main_files/figure-html/unnamed-chunk-10-1.png" rel="nofollow noopener">http://mdatools.com/mdatools/_main_files/figure-html/unnamed-chunk-10-1.png</a></p>
<p>No problem. Now raw file in the other repo:</p>
<p><a href="https://github.com/svkucheryavski/mdatools.bookdown/blob/master/docs/_main_files/figure-html/unnamed-chunk-10-1.png" rel="nofollow noopener">https://github.com/svkucheryavski/mdatools.bookdown/blob/master/docs/_main_files/figure-html/unnamed-chunk-10-1.png</a></p>
<p>Same name, same location, everything (except the repository name of course) is the same, however if I try to get it via pages I got 404:</p>
<p><a href="http://mdatools.com/mdatools.bookdown/_main_files/figure-html/unnamed-chunk-10-1.png" rel="nofollow noopener">http://mdatools.com/mdatools.bookdown/_main_files/figure-html/unnamed-chunk-10-1.png</a></p>
<p>Spend whole day reading forum, will be greatful for any help! I use <code>.nojekyll</code> in both cases.</p>

**Comment by svkucheryavski** [comment, marked as answer]
<p>Solved — file <code>.nojekyll</code> should be placed to the repository with main Github Pages content. Otherwise folders starting with underscore are not processed correctly. </p>

**Reply by srbhp** [reply]
Just spend a whole day to figure this out. Thanks for putting it here.
